deploy/base/metrics_client.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 rules:
 - nonResourceURLs:
   - /metrics
```

Comment:
Is this still needed?

Reply:
Yep, we need both paths to be accessible.